src/frontend/src/pages/MainPage/components/componentsComponent/index.tsx

```diff
@@ -55,7 +52,7 @@ export default function ComponentsComponent({
     (state) => state.selectedFlowsComponentsCards,
   );
 
-  const [handleFileDrop] = useFileDrop(uploadFlow, type)!;
+  const [handleFileDrop] = useFileDrop(type)!;
```

Comment:
1 - there is no reason for the ! mark in that line
2 - why is the useFileDrop returning a list if there is only one element on it?